package/Dockerfile-windows.agent

```diff
@@ -0,0 +1,26 @@
+ARG RELEASES
+ARG SERVERCORE_VERSION
+ARG VERSION
+FROM mcr.microsoft.com/windows/servercore:${SERVERCORE_VERSION}
+SHELL ["powershell", "-NoLogo", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+RUN pushd c:\; \
+    $URL = 'https://github.com/git-for-windows/git/releases/download/v2.29.2.windows.3/MinGit-2.29.2.3-64-bit.zip'; \
+    \
+    Write-Host ('Downloading git from {0} ...' -f $URL); \
+    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; \
+    Invoke-WebRequest -UseBasicParsing -OutFile c:\git.zip -Uri $URL; \
+    \
+    Write-Host 'Expanding ...'; \
+    Expand-Archive -Force -Path c:\git.zip -DestinationPath c:\git\.; \
+    \
+    Write-Host 'Cleaning ...'; \
+    Remove-Item -Force -Recurse -Path c:\git.zip; \
+    \
+    Write-Host 'Complete.'; \
+    popd;
+RUN $URL = 'https://{0}/fleet/{1}/fleetagent-windows-amd64.exe' -f $env:RELEASES, $env:VERSION; \
```

Comment:
@luthermonson: Am I pulling in the `ARG` correctly here? I don't think so. We may just be able to use `-f ${RELEASES}, ${VERSION};`. Thoughts?

Reply:
Double-checked the docs. What we have is good!